bin/CodeChecker.py

```diff
@@ -121,7 +121,14 @@ def signal_handler(signum, frame):
         # extend the system argument list with these options and try to parse
         # the argument list again to validate it.
         if 'func_process_config_file' in args:
-            cfg_args = args.func_process_config_file(args)
+            if len(sys.argv) > 1:
+                called_sub_command = sys.argv[1]
+
+            # The subcommand name is analyze but the
+            # configuration section name is analyzer.
+            if called_sub_command == 'analyze':
```

Comment:
This should be handle inside the `func_process_config_file` function.

Maybe we can change the section name to `analyze` (in config file, in user guide etc.) too and we can keep the below condition for backward compatibility.